reference/customers.v3.yml

```
@@ -3292,10 +3295,12 @@ components:
           description: The country name of the customer address.
           type: string
         form_fields:
-          description: Array of form fields. Controlled by `formfields` parameter.
-          type: array
-          items:
-            $ref: '#/components/schemas/formFieldValue_Full'
+                description: Array of form fields. Controlled by `formfields` parameter.
+                type: array
+                items:
+                  allOf:
+                    - $ref: "#/components/schemas/formFieldValue_CustomerAddress"
```

```suggestion
                    - $ref: "#/components/schemas/formFieldValue"
```
https://github.com/bigcommerce/api-specs/pull/699/files#diff-e32e9f10280459c2f2dd1cd85354d06cc59a229a38846daf423e065b81a5b25fR3609